config/index.js

```diff
@@ -52,7 +52,8 @@ const config = {
       environment: process.env.ROLLBAR_ENV
     },
     port: process.env.PORT,
-    url: process.env.JURL
+    url: process.env.JURL,
+    expiration: process.env.EXP
```

Comment:
Put a default in expiration time something like `process.env.EXP_TOKEN_SECONDS || 1000`, be more declarative with env variables. Only with read them you have to know what does and how, `process.env.EXP_TOKEN_SECONDS` -> Set expiration time of token in seconds